packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -507,6 +507,8 @@ export type SchedulerJobEvent = BaseTrack & {
         jobId: string;
         schedulerId: string;
         schedulerTargetId: string;
+        task?: 'sendEmailNotification' | 'sendSlackNotification';
+        type?: string;
```

Comment:
We could get this information by joining the information from the scheduler event that already has the `resourceType: 'dashboard' | 'chart';` and the target type `type: 'slack' | 'email';`.
If @TuringLovesDeathMetal finds it useful to have this data here, we should make it required and use the same terminology as the other events. Let me know what you think @rephus 
```
resourceType: 'dashboard' | 'chart';
type: 'slack' | 'email';
```